### PR TITLE
 scan-for-vulnerabilities.md

### PR DESCRIPTION
**Pull request by yelghali**
@dubrie

### CODE REVIEW

File: docs/catalog/cloud/scan-for-vulnerabilities.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+version: 1.0
+submitted_by: yelghali
+published_date: tbd
+category: cloud
+tags: 
+- security
```
Comment (dubrie) [inline docs/catalog/cloud/scan-for-vulnerabilities.md:7]: please include `cloud` in the list of tags as well

File: docs/catalog/cloud/scan-for-vulnerabilities.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+- role:cloud-engineer
+- size:medium
+---
+
+# Scan for vulnerabilities
+
+## Description
+Many attacks on cloud infrastructure seek to misuse deployed resources for the attacker's direct gain leading to an unnecessary spike in usage and cost. 
+
+Vulnerability scanning and EDR tools help minimize the window of opportunity for attackers and mitigate any potential malicious usage of resources.
```
Comment (dubrie) [inline docs/catalog/cloud/scan-for-vulnerabilities.md:19]: Can you define what EDR stands for here?

File: docs/catalog/cloud/scan-for-vulnerabilities.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+## SCI Impact
+`SCI = (E * I) + M per R`
+[Software Carbon Intensity Spec](https://grnsft.org/sci)
+
+Regarding the SCI equation, scanning for vulnerabilities will impact:
+
+- `M`: By preventing heavy misusage of resources, we reduce unnecesseary embodied carbon emissions.
+
+## Assumptions
+Zero Trust security policy is implemented, which covers security controls for infrastructure, applications, data, networking, RBAC and devices.
```
Comment (dubrie) [inline docs/catalog/cloud/scan-for-vulnerabilities.md:33]: can you define what RBAC stands for here?

**Review by dubrie — CHANGES_REQUESTED** [review]
some changes requested from the initial review. Additionally, if this was copied from somewhere else on the web, please use the References section to link back to the original source for proper citation and linking

**Comment by yelghali** [comment]
ok

**Review by dubrie — APPROVED** [review]

**Comment by dubrie** [comment]
Looks great, thank you!

**Pull request merged**